tests/adsb_test.py

```diff
@@ -0,0 +1,63 @@
+from pyModeS import adsb
```

Comment:
With addition of [tox](https://tox.readthedocs.io/en/latest/) we won't have this issue when running tests. For each version of python specified in the `envlist` found in `tox.ini`, tox will create a virtualenv, install all the dependencies need for testing specified by `deps`, and run the `commands` to test the code. Checkout what it looks like [here](https://asciinema.org/a/dsf103y9x61666u2633bc1bdq).

Let me know what you think